packages/manager/apps/dedicated/client/app/index.html

```diff
@@ -92,29 +92,6 @@
             </div>
         </div>
 
-        <div data-ng-controller="UserContractsCtrl" class="ng-cloak">
```

Comment:
I don't see the removal of this controller and associated file, is that wanted ?

Reply:
I have removed this, its not required